libp2p/7-properties.md

```diff
@@ -190,3 +190,19 @@ For now, we use [protobuf](https://github.com/google/protobuf) for all protocol
 ### 7.5.6 Secure Communications
 
 The wire protocol is -- of course -- wrapped with encryption. We use cyphersuites similar to TLS. This is explained further in the [network spec](./#encryption).
+
+### 7.5.7 Protocol Multicodecs
+
+Here, we present a table with the multicodecs defined for each IPFS protocol that has a wire componenent. This list may change overtime and currently exists as a guide for implementation
+
+| protocol | multicodec |
+| :------: | :--------: |
+| secio    | /ipfs/secio/1.0.0 |
+| plain    | /plaintext/1.0.0 |
+| identify | /ipfs/identify/1.0.0 |
+| multistream | /ipfs/multistream/
+| yamux | /yamux/1.0.0 |
+| multiplex | /multiplex/6.7.0 |
+| spdy | /spdy/3.1.0 |
+| dht | /ipfs/dht/1.0.0 | 
+| bitswap | /ipfs/bitswap/1.0.0 |
```

Comment:
Hrm... that appears to be a typo. I remember making a commit where I added versions to this, diag and the dht. Where did that go?